goldenmaster/Plugins/TbEnum/Source/TbEnum/Private/Generated/Monitor/TbEnumFactory.cpp

```diff
@@ -72,17 +72,33 @@ TScriptInterface<ITbEnumEnumInterfaceInterface> createTbEnumEnumInterface(UGameI
 
 TScriptInterface<ITbEnumEnumInterfaceInterface> FTbEnumModuleFactory::createITbEnumEnumInterfaceInterface(UGameInstance* GameInstance, FSubsystemCollectionBase& Collection)
 {
-	UTbEnumSettings* settings = GetMutableDefault<UTbEnumSettings>();
+	UTbEnumSettings* TbEnumSettings = GetMutableDefault<UTbEnumSettings>();
 
-	switch (settings->ServiceConnection)
+	// first check whether local backend is requested
+	if (TbEnumSettings->ConnectionIdentifier == "Local")
 	{
-	case ETbEnumConnection::CONNECTION_OLINK:
-		return createTbEnumEnumInterfaceOLink(GameInstance, Collection);
-	case ETbEnumConnection::CONNECTION_LOCAL:
-		return createTbEnumEnumInterface(GameInstance, Collection);
-	default:
 		return createTbEnumEnumInterface(GameInstance, Collection);
 	}
+
+	// otherwise iterate over known connections and check whether applicable
+	UApiGearSettings* ApiGearSettings = GetMutableDefault<UApiGearSettings>();
+	for (auto ConnectionSetting : ApiGearSettings->Connections)
```

Comment:
this could be sdtI'd change to:
	auto ApiGearConfiguredConnections = GetMutableDefault<UApiGearSettings>()->Connections;

	auto requstedConnection = std::find_if(ApiGearConfiguredConnections.begin(), ApiGearConfiguredConnections.end(),
		[&TbEnumSettings](auto &ConnectionSetting) {
			ConnectionSetting.Key == TbEnumSettings->ConnectionIdentifier
		});

	auto connectionConfigured = requstedConnection != ApiGearConfiguredConnections.end();
	// Other protocols not supported. To support it edit templates:
	// add protocol handler class for this interface like createTbEnumEnumInterfaceOLink and other necessary infrastructure
	// extend this function in templates to handle protocol of your choice
	if (connectionConfigured && requstedConnection->Value.ProtocolIdentifier == "olink")
	{
		return createTbEnumEnumInterfaceOLink(Collection);
	}
	
I think the variables names are helpful here more than comments, also extended a comment regarding not supporting other protocols
Plus this could use a constant for "olink"